### PR TITLE
Grey thumbnails + added missing thumbnails

### PR DESCRIPTION
**Pull request by apepper**
Design by @agessler.

obj thumbnails
widget thumbnails

### CODE REVIEW

File: src/assets/images/blog_obj.svg [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
+            <rect id="Rectangle-11" stroke="#aaaaaa" stroke-linecap="square" x="82.5" y="21.5" width="45" height="27"></rect>
+            <rect id="Rectangle-11" stroke="#aaaaaa" stroke-linecap="square" x="82.5" y="55.5" width="45" height="27"></rect>
+            <rect id="Rectangle" fill="#aaaaaa" x="88" y="25" width="32" height="4"></rect>
+            <rect id="Rectangle" fill="#aaaaaa" x="88" y="59" width="32" height="4"></rect>
+            <path d="M88.375,33 L114.625,33" id="Line" stroke="#aaaaaa" stroke-linecap="square"></path>
+            <path d="M88.375,67 L114.625,67" id="Line" stroke="#aaaaaa" stroke-linecap="square"></path>
+            <path d="M88.38,37 L106.62,37" id="Line" stroke="#aaaaaa" stroke-linecap="square"></path>
+            <path d="M88.38,71 L106.62,71" id="Line" stroke="#aaaaaa" stroke-linecap="square"></path>
+            <path d="M88.3709677,41 L110.629032,41" id="Line" stroke="#aaaaaa" stroke-linecap="square"></path>
+            <path d="M88.3709677,75 L110.629032,75" id="Line" stroke="#FF6aaaaaaB6B" stroke-linecap="square"></path>
```
Comment (dcsaszar) [inline src/assets/images/blog_obj.svg:18]: * [ ] 🙀 AAAAAA!

File: src/assets/images/link_widget.svg [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 150 100" style="enable-background:new 0 0 150 100;" xml:space="preserve">
+<style type="text/css">
+	.st0{enable-background:new    ;}
```
Comment (dcsaszar) [inline src/assets/images/link_widget.svg:6]: * [ ] FYI according to https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/enable-background `enable-background` is deprecated.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Mostly fine. I didn't check each change, only scanned over them. See comments, I found at least one error.

* [ ] FYI Regarding the colors I personally would prefer a version with higher contrast. The former color icons provided at least a color contrast, which made them stand out more than grey on white.

File: src/assets/images/link_widget.svg (hunk 2 above)
Comment (agessler) [reply src/assets/images/link_widget.svg:6]: @dcsaszar This is not my fault.. talk to Adobe.

File: src/assets/images/blog_obj.svg (hunk 1 above)
Comment (agessler) [reply src/assets/images/blog_obj.svg:18]: Copy Paste Issue.. I fix

**Comment by agessler** [comment]
@apepper FIXES HERE: 
https://github.com/agessler/scrivito_example_app_js/commit/5532409a6822bf3f1b05accb67435923b871850e

**Comment by agessler** [comment]
@dcsaszar Yes the former color is more notably, but does not fit to our darker side panel. The new color fit to both worlds. I real live you will see more screenshots - so there is no issue.

**Comment by apepper** [comment]
REVIEW (990b1bc) /cc @dcsaszar

**Review by dcsaszar — APPROVED** [review]

**Pull request merged**